### PR TITLE
Export labeling classes

### PR DESCRIPTION
**Pull request by bwasty**
...for use in `webgl-examples`

### CODE REVIEW

File: source/webgl-operate.ts [hunk 1]
```diff
@@ -13,6 +13,13 @@ export import fetch = root_fetch;
 import * as root_raymath from './raymath';
 export import ray_math = root_raymath;
 
+export { FontFace } from './fontface';
+export { FontLoader } from './fontloader';
+export { GlyphVertex, GlyphVertices } from './glyphvertices';
+export { Label } from './label';
+export { LabelGeometry } from './LabelGeometry';
```
Comment (anne-gropler) [inline source/webgl-operate.ts]: I changed the file name using lower case, as you suggested, so this line needs to be edited.

Comment (bwasty) [reply source/webgl-operate.ts]: done

**Review by anne-gropler — APPROVED** [review]

**Comment by anne-gropler** [comment]
@bwasty Question: Why is it needed to export those labeling classes this way, while the testrenderer in webgl-examples can use e.g. Texture2 without exporting Texture2 like the labeling classes?

**Comment by bwasty** [comment]
@anne-gropler `Texture2` is also exported - in `webgl-operate.slim.ts`, which is included in `webgl-operate.ts` (first line).

**Comment by anne-gropler** [comment]
Well yes, my point is: why do we export the labeling classes *differently*, i.e. *not* in `webgl-operate.slim.ts`?

**Comment by bwasty** [comment]
Well, `slim` is supposed to be smaller and only the core as far as I understand, and I think labeling counts as 'advanced' and is not needed by every webgl-operate based app.

**Pull request merged**